Source/Particles/Resampling/VelocityCoincidenceThinning.H

```diff
@@ -191,6 +208,7 @@ public:
         amrex::ParticleReal dur, dutheta, duphi;
         amrex::ParticleReal dux, duy, duz;
         amrex::ParticleReal ux_min, uy_min, uz_min, ux_max, uy_max;
+        amrex::ParticleReal m_max_w;
```

Comment:
Keeping with convention of the other member variables (and since `velocityBincalculator` is a `struct`, this might instead be
```suggestion
        amrex::ParticleReal max_weight;
```